packages/paste-core/components/avatar/src/types.ts

```diff
@@ -6,15 +6,60 @@ import type { HTMLPasteProps } from "@twilio-paste/types";
 export type ColorVariants = "default" | "decorative10" | "decorative20" | "decorative30" | "decorative40";
 export type AvatarVariants = "user" | "entity";
 
-export type AvatarProps = HTMLPasteProps<"div"> &
-  Pick<BoxProps, "element"> & {
-    name: string;
-    size?: IconSize;
-    icon?: React.FC<React.PropsWithChildren<GenericIconProps>>;
-    src?: string;
-    color?: ColorVariants;
-    variant?: AvatarVariants;
-  };
+export type AvatarProps = HTMLPasteProps<"div"> & {
+  /**
+   * Used to determine the initials, alt text (if using an image), or title (if using an icon) of the Avatar.
+   *
+   * @type {string}
+   * @memberof AvatarProps
+   */
+  name: string;
+  /**
+   *
```

Comment:
question: noticed a few of these tsdoc comments without description text. not sure if intentional, seeking clarity

Reply:
It's intentional - I think it's fine to not provide descriptions for self-explanatory or commonly used props. I was also going off of prior tsdoc comments of Simon's PRs which don't all include descriptions / defaults / etc.

Reply:
Yeah I'm just replicating what the current docs do right now, just to migrate. If they weren't documented with a description I didn't bother creating a new one